tech-far-hub/tsconfig.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Is this how the tsconfig came out of the box? With every option available to uncomment? 🤯 

If so, that's so cool!

Reply:
It is! And until I know what I'm doing, I'm leaving it lol!